resources/views/auth/login.blade.php

```diff
@@ -8,103 +8,127 @@
-				return false;
-			}
-			// let auth2;
-			gapi.load('auth2', function() {
-				var auth2 = gapi.auth2.init({
-					hosted_domain: 'itesm.mx',
-					prompt: 'select_account'
-				});
-				auth2.then(function() {
-					const hasUser = <?= Auth::user() === NULL ? 'false' : 'true' ?>;
-					if(auth2.isSignedIn.get() && !hasUser) {
-						auth2.signOut().then(function() {
-							loadBtn();
-						});
-					}
-					else {
-						setTimeout(loadBtn, 1200);
-					}
-				});
-			});
+		$img.on('error', function() {
+			loadPage(false);
+		});
+		$img.css("display", "none").appendTo(document.body);
+	});
+
+	function loadPage(isContentBlockingEnabled) {
+		if(!isContentBlockingEnabled) {
+			let msg = 'Please disable content blocking on this page to be able to sign in.';
+			console.error(msg);
+			$('#my-signin2').html(msg);
+			return false;
 		}
-		function loadBtn() {
-			gapi.signin2.render('my-signin2', {
-				'width': 240,
-				'height': 50,
-				'longtitle': true,
-				'theme': 'light',
-				'onsuccess': onSignIn,
-				'onfailure': onFailure
+		// let auth2;
+		gapi.load('auth2', function() {
+			var auth2 = gapi.auth2.init({
+				hosted_domain: 'itesm.mx',
+				prompt: 'select_account'
+			});
+			auth2.then(function() {
+				const hasUser = <?= Auth::user() === NULL ? 'false' : 'true' ?>;
+				if(auth2.isSignedIn.get() && !hasUser) {
+					auth2.signOut().then(function() {
+						loadBtn();
+					});
+				}
+				else {
+					setTimeout(loadBtn, 1200);
+				}
 			});
+		});
+	}
+
+	function loadBtn() {
+		gapi.signin2.render('my-signin2', {
+			'width': 240,
+			'height': 50,
+			'longtitle': true,
+			'theme': 'light',
+			'onsuccess': onSignIn,
+			'onfailure': onFailure
+		});
+	}
+
+	function showError(errorMessage) {
+		$('#errorReason').html(errorMessage);
+		$('#signInError').removeClass("hidden");
+	}
+
+	function onFailure(e) {
+		// Handle the error where user does not match domain
+		if(e.error) {
+			showError(errorMessageItesmAccount);
 		}
-		function onFailure(e) {
-			// Handle the error where user does not match domain
-			if(e.error) {
-				console.error(e);
-			}
-			if(e.type && e.type === "tokenFailed") {
-				alert('Could not sign you in using this account. Please use your \@itesm.mx account');
-			}
+		if(e.type && e.type === "tokenFailed") {
+			showError(errorMessageGeneric);
 		}
-		function onSignIn(googleUser) {
-			var profile = googleUser.getBasicProfile();
-			let token = googleUser.getAuthResponse().id_token;
-			$('#my-signin2').fadeOut();
-			// Send the request now
-			$.ajax({
-				url: '/login',
-				method: 'POST',
-				headers: {
-					'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
-				},
-				data: {'id_token': token},
-				dataType: 'json',
-				success: function(data) {
-					if(data && data.success) {
+	}
+
+	function onSignIn(googleUser) {
+		var profile = googleUser.getBasicProfile();
+		let token = googleUser.getAuthResponse().id_token;
+		// $('#my-signin2').fadeOut();
```

Comment:
Por qué quitaste el fadeOut?

Reply:
No estoy muy segura de para qué lo necesitamos. El login sólo se muestra si no estás signed in. ¿Para qué lo queremos quitar?

Reply:
Para que no le piquen una y otra vez

Reply:
qué visionario. will fix.